recipes/mage-ai/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "mage-ai" %}
+{% set version = "0.4.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6d99fbeb720db9b3d6b643e4197bb6b07463c40e03a4bc750b201ac9162f7eda
+
+build:
+  skip: True  # [win]
+  skip: True  # [py<37]
```

Why are you skipping Windows? It looks like this can be noarch.
```suggestion
  noarch: python
```